src/components/CarouselGrid/CarouselGrid.js

```diff
@@ -0,0 +1,58 @@
+import React from "react"
+import PropTypes from "prop-types"
+import styles from './CarouselGrid.module.scss'
+import Carousel from '../Carousel/Carousel'
+
+
+
+export default class CarouselGrid extends React.Component {
+
+
+    constructor(props) {
+        super(props)
+        this.state = {
+
+        }
+
+
+    }
+
+
+    static propTypes = {
+        gridData: PropTypes.array.isRequired,
+        gridName: PropTypes.string.isRequired,
+
+    }
+
+
+
+    render() {
+        return (
+
+            <div className={styles.styleCarrousel} >
+                {/* {console.log(this.props.gridData)} */}
+                {this.props.gridData.map((item, index) => (
+
+                    <Carousel
+                        className={styles.imageSmall}
+                        carouselName={this.props.gridName}
+                        carouselKeyIndex={index}
```

Comment:
This index as a key is not the best way if we can help it https://reactjs.org/docs/lists-and-keys.html#keys